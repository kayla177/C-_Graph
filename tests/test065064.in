ENTITY RFH2LFC3 name_RFH2 author
ENTITY XDFSGHR7 name_XDFS institution
ENTITY B8P7R80V name_B8P7 institution
ENTITY 044P4S4G name_044P institution
ENTITY 2F45YBQV name_2F45 journal
ENTITY OBGZ5S75 name_OBGZ conference
ENTITY CDEI3F4O name_CDEI paper
ENTITY TFS38VWL name_TFS3 institution
ENTITY D1X2WJOP name_D1X2 paper
ENTITY TTAPO30G name_TTAP conference
RELATIONSHIP XDFSGHR7 presented_at OBGZ5S75 4.5
RELATIONSHIP XDFSGHR7 presented_at 2F45YBQV 4.0
RELATIONSHIP CDEI3F4O collaborated_with RFH2LFC3 9.2
RELATIONSHIP XDFSGHR7 collaborated_with CDEI3F4O 2.9
RELATIONSHIP OBGZ5S75 affiliated_with TTAPO30G 0.4
RELATIONSHIP D1X2WJOP affiliated_with TFS38VWL 3.4
RELATIONSHIP XDFSGHR7 cited_by 044P4S4G 1.6
RELATIONSHIP RFH2LFC3 cited_by 044P4S4G 0.9
RELATIONSHIP RFH2LFC3 cited_by 2F45YBQV 2.6
RELATIONSHIP B8P7R80V collaborated_with 2F45YBQV 6.5
RELATIONSHIP TFS38VWL presented_at B8P7R80V 7.2
RELATIONSHIP TFS38VWL authored_by TTAPO30G 3.5
RELATIONSHIP D1X2WJOP affiliated_with CDEI3F4O 4.5
RELATIONSHIP TFS38VWL cited_by CDEI3F4O 1.3
RELATIONSHIP D1X2WJOP collaborated_with TTAPO30G 7.8
PATH RFH2LFC3 044P4S4G
PATH D1X2WJOP CDEI3F4O
PATH XDFSGHR7 044P4S4G
PRINT 2F45YBQV
HIGHEST
DELETE 2F45YBQV
PATH TFS38VWL RFH2LFC3
HIGHEST
HIGHEST
PATH TFS38VWL 044P4S4G
EXIT
