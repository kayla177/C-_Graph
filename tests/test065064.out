success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
RFH2LFC3 CDEI3F4O XDFSGHR7 044P4S4G 13.7
D1X2WJOP CDEI3F4O 4.5
XDFSGHR7 044P4S4G 1.6
XDFSGHR7 RFH2LFC3 B8P7R80V
OBGZ5S75 TFS38VWL 22.2
success
TFS38VWL D1X2WJOP CDEI3F4O RFH2LFC3 17.1
RFH2LFC3 B8P7R80V 24.3
RFH2LFC3 B8P7R80V 24.3
TFS38VWL D1X2WJOP CDEI3F4O XDFSGHR7 044P4S4G 12.4
